2nd-gen/alerts-to-discord/functions/index.js

```diff
@@ -139,3 +142,64 @@ UDID **${testerDeviceIdentifier}** for ${testerDeviceModelName}
   }
 });
 // [END v2Alerts]
+
+/**
+ * Function triggered by Firebase Performance that publishes
```

Comment:
Update product name to "Firebase Performance Monitoring"

Reply:
Updated.